### ISSUE TITLE
Undesirable glow effect on dark scenes (HLS)

### ISSUE THREAD

**Issue by awakenedguy** [flowplayer/flowplayer#1296]
Hi,

I am using flowplayer 7.2.1, encrypted HLS, splash setup, Windows 10, Chrome 62.0.3202.89 (official build), (64 bit).

Here is 3 minutes sample when this sometimes psychodelic effect can be observed. The video is encoded using manually crafted ffmpeg parameters (latest nightly build from 9 Nov 2017). No ffmpeg filters are used:
http://gdf53zgt.b-cdn.net/cut/index.html

Take a look at screenshots. Same sample is perfectly good when played offline with other players:
flowplayer (dark glow effect):
http://prntscr.com/h8r51c

PotPlayer (good):
http://prntscr.com/h8r5u6

VLC player (good):
http://prntscr.com/h8r6ze

And one more frame screenshot where it is particularly visible:
flowplayer (dark glow effect):
http://prntscr.com/h8r847

PotPlayer (good):
http://prntscr.com/h8r8on

VLC player (good):
http://prntscr.com/h8r9hp

Is there a way to fix it? Thanks.

**Comment by bbbo** [comment]
Sorry, cannot reproduce. Looks fine for me in Win10 and Chrome 62. Do you see the same effect in a plain video tag and hls.js , or the native hsl.js  [Chrome extension](https://chrome.google.com/webstore/detail/native-hls-playback/emnphkkblegpebimobpbekeedfgemhof) ?

![2017-11-10 18_58_04-windows 10](https://user-images.githubusercontent.com/1395672/32672230-976730fe-c64a-11e7-821a-3049108962fa.png)

**Comment by awakenedguy** [comment]
Checked my link from the phone - indeed, everything is fine.

But I just found the problem, it is my local one. Actually I am using a laptop with 2 external screens attached:

1st one via HDMI
2nd one via external DisplayLink adapter, namely this one:
https://www.startech.com/AV/USB-Video-Adapters/usb-3-4k-displayport-adapter~USB32DP4K

It is necessary because the screen is 2560x1440 and there is no way to get that resolution with this laptop without this adapter. It is otherwise limited to 1920x1080.

So when this hi resolution screen is on via this adapter, the bug is observed on both external screens.

When I plug it out and only leave normal HDMI external screen, the problem is gone.

So Windows DisplayLink drivers are to blame.
